cmd/jujud/agent/caasoperator/manifolds.go

```diff
@@ -130,6 +130,7 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			APIOpen:              api.Open,
 			APIConfigWatcherName: apiConfigWatcherName,
 			NewConnection:        apicaller.OnlyConnect,
+			Logger:               loggo.GetLogger("juju.worker.apicaller"),
```

Comment:
So this one will still be a package level one? Oh, I guess this worker isn't model specific?